diff --git a/singletons/announcement_manager.gd b/singletons/announcement_manager.gd
index cdd7d8c..351afbd 100644
--- a/singletons/announcement_manager.gd
+++ b/singletons/announcement_manager.gd
@@ -17,6 +17,8 @@ var _image_to_download: String
 
 
 func _ready() -> void :
+	if OS.get_name() == "HTML5":
+		return
 	if OS.get_name().begins_with("Seaven"):
 		return
 
diff --git a/singletons/crash_reporter.gd b/singletons/crash_reporter.gd
index 98e7358..f7583ad 100644
--- a/singletons/crash_reporter.gd
+++ b/singletons/crash_reporter.gd
@@ -10,6 +10,8 @@ var previous_crashed_mod: = ""
 var _mod_unzip_error: = "The mod zip at path"
 
 func _init() -> void :
+	if OS.get_name() == "HTML5":
+		return
 	
 	print(_version_string())
 	
diff --git a/addons/mod_loader/mod_loader_store.gd b/addons/mod_loader/mod_loader_store.gd
index 4ecb1c5..66e00a4 100644
--- a/addons/mod_loader/mod_loader_store.gd
+++ b/addons/mod_loader/mod_loader_store.gd
@@ -137,6 +137,10 @@ var ml_options: = {
 
 
 func _init():
+	if OS.get_name() == "HTML5":
+		ml_options.enable_mods = false
+		is_initializing = false
+		return
 	_update_ml_options_from_options_resource()
 	_update_ml_options_from_cli_args()
 	
diff --git a/addons/mod_loader/mod_loader.gd b/addons/mod_loader/mod_loader.gd
index 75c8934..f291eed 100644
--- a/addons/mod_loader/mod_loader.gd
+++ b/addons/mod_loader/mod_loader.gd
@@ -38,7 +38,9 @@ var mod_data: = {} setget , deprecated_direct_access_mod_data
 
 
 
 func _init() -> void :
+	if OS.get_name() == "HTML5":
+		return
 	
 	_check_autoload_positions()
 
@@ -69,6 +71,8 @@
 
 
 func _ready():
+	if OS.get_name() == "HTML5":
+		return
 	
 	
 	if not ModLoaderStore.user_profiles.has("default"):
